### DISCUSSION TITLE
Why hPyT is not working with pyinstaller  converting

### DISCUSSION THREAD

**Discussion by Minty73** [Zingzy/hPyT#39] [Q&A] [answered]
Can you fix this?

**Comment by Zingzy** [comment]
Hello, can you please share the errors which are being showed? Can you also please share the command you used for packing your app?

**Reply by Zingzy** [reply]
I have tried to build applications which has hPyT as a dependency using pyinstaller, and I was able to bundle it without any problems.

**Reply by Minty73** [reply]
Command: pyinstaller --noconfirm --onefile --windowed --icon "C:\Users\User\Desktop\YT\text.ico"  "C:\Users\User\Desktop\YT\txtedit.pyw"
![Без имени](https://github.com/user-attachments/assets/61dd5230-3afd-432c-8a30-0fcf4cba723e)

**Reply by Zingzy** [reply, marked as answer]
Hello @Minty73, I think you are not using the latest version of hPyT (v1.3.6)... If you upgrade to the latest version, this error might not occur.

**Reply by Minty73** [reply]
Thank you

**Reply by jttaggart** [reply]
I'm using 1.4.0 with PyInstaller and everything is working. Great solution for handling titles.

**Reply by Zingzy** [reply]
Glad to know, thank you @jttaggart
